__tests__/integration/query-service-points.test.ts

```diff
@@ -13,10 +15,11 @@ it('resolves serivePoints', async () => {
           details {
             isDefaultForCampus
             isDefaultPickup
+            notes
           }
         }
       }`;
-
+      
```

Comment:
Would it be worth it to add notes to one of the mocked service point responses  to test that that info is also parsed out correctly?

Reply:
This is tested here:
https://github.com/sul-dlss/folio-graphql/blob/88c8f6f731bf1bb177c261d59acd7b593151fe1e/__tests__/service-points-api.test.ts#L32

But If you think putting a notes check in this integration test is adding more coverage I am happy to

Reply:
I think it may be useful to have one of the examples have notes, since you're already testing the other two fields under details.

Reply:
I don't think it's required though, as it is being covered elsewhere, so I'm happy to approve this.